tests/test_create.py

```diff
@@ -404,6 +405,35 @@ def test_clone_offline_multichannel_with_untracked(self):
                     assert_package_is_installed(clone_prefix, 'rpy2')
                     assert isfile(join(clone_prefix, 'condarc'))  # untracked file
 
+    @pytest.mark.timeout(300)
+    def test_channel_order_channel_priority_true(self):
+        with make_temp_env("python=3 pycosat==0.6.1") as prefix:
+            assert_package_is_installed(prefix, 'python')
+            assert_package_is_installed(prefix, 'pycosat')
+
+            # add conda-forge channel
+            run_command(Commands.CONFIG, prefix, "--add channels conda-forge")
+            from conda.config import get_rc_urls
+            assert get_rc_urls() == ['conda-forge', 'defaults']
+
+            # update --all
+            update_stdout, _ = run_command(Commands.UPDATE, prefix, '--all')
+
+            # pycosat should be in the SUPERCEDED list
+            superceded_split = update_stdout.split('SUPERCEDED')
+            assert len(superceded_split) == 2
+            assert 'pycosat' in superceded_split[1]
+
+            # python sys.version should show conda-forge python
+            sys_version = check_output('{0} -c "import sys; print(sys.version)"'
+                                       ''.format(join(prefix, PYTHON_BINARY)), shell=True)
+            sys_version = sys_version.decode('utf-8')
+            assert "packaged by conda-forge" in sys_version, sys_version
```

Comment:
@kalefranz , the test fails in this line 

    assert "packaged by conda-forge" in sys_version, sys_version

The python is still in default channel, seems we need to activate env to work. Maybe we can use the get_conda_list_tuple method, and check the 4th element is 'conda-forge'. Just like how we check the version of pycosat.